common/saber_base.h

```diff
@@ -303,7 +303,7 @@ public:                                                         \
     if (mode == COLOR_CHANGE_MODE_NONE) {
       DoChange(EXIT_COLOR_CHANGE);
     } else {
-      DoChange(ENTER_COLOR_CHANGE);
+      if (mode != COLOR_CHANGE_MODE_ZOOMED) DoChange(ENTER_COLOR_CHANGE);
```

Comment:
This doesn't seem right.
I think we need to look at what color_change_mode_ was before and call DoChange(ENTER_COLOR_CHANGE) if color_change_mode_ was COLOR_CHANGE_MODE_NONE before.

Reply:
Is this what you were thinking?
Currently in my prop, since we have two types of ColorZoom (COLOR_CHANGE_MODE_ZOOMED & Color Editing), I handle the sound for changing modes in the prop itself, not sure if other props will need transitions to other modes might be simpler to leave that on the prop side.